### DISCUSSION TITLE
How to generate new fields?

### DISCUSSION THREAD

**Discussion by adrian-ub** [filamentphp/filament#31] [Help] [answered]
Hi devs, I am very interested in this package and I would like if you can create new fields like in Laravel Nova?

**Comment by danharrin** [comment, marked as answer]
Yes! You may extend the `Filament\Forms\Components\Field` class and overwrite the `$view` property with the path to a custom view. I will document the process soon and keep you updated.

**Reply by ryangjchandler** [reply]
@adrian-ub You can upgrade to latest version and use the `make:filament-field` command too now.
